CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Changed
 - **slick slider** Integrate with vtex.storecomponents/Slider
+### Fixed
+- Fix banner link working.
```

Comment:
Fix bug of banner link not working

Reply:
Fix issue that makes banner link not working properly 💃